app/javascript/tests/public_health/PatientsTable.test.js

```diff
@@ -0,0 +1,152 @@
+    });
+  });
+
+  it('Inputting text into search bar calls update table function', () => {
+    const wrapper = getExposureWrapper();
+    const handleSearchChangeSpy = jest.spyOn(wrapper.instance(), 'updateTable');
+    expect(handleSearchChangeSpy).toHaveBeenCalledTimes(0);
+    expect(wrapper.state('query').search).toEqual('');
+    wrapper.find('#search').simulate('change', { target: { id: 'search', value: 'search' } });
+    expect(wrapper.state('query').search).toEqual('search');
+    expect(handleSearchChangeSpy).toHaveBeenCalledTimes(1);
+  });
+
+  it ('Clicking "Close Records" option displays Close Records modal', () => {
+    const wrapper = getExposureWrapper();
+    expect(wrapper.find(CloseRecords).exists()).toBeFalsy();
+    expect(wrapper.find(Dropdown.Item).at(0).text().includes(dropdownOptions[0])).toBeTruthy();
+    wrapper.find(Dropdown.Item).at(0).simulate('click');
+    expect(wrapper.find(CloseRecords).exists()).toBeTruthy();
+  });
+
+  it ('Clicking "Update Case Status" option displays Update Case Status modal', () => {
+    const wrapper = getExposureWrapper();
+    expect(wrapper.find(UpdateCaseStatus).exists()).toBeFalsy();
+    expect(wrapper.find(Dropdown.Item).at(1).text().includes(dropdownOptions[1])).toBeTruthy();
+    wrapper.find(Dropdown.Item).at(1).simulate('click');
+    expect(wrapper.find(UpdateCaseStatus).exists()).toBeTruthy();
+  });
+
+  it ('Clicking "Update Assigned User" option displays Update Assigned User modal', () => {
+    const wrapper = getExposureWrapper();
+    expect(wrapper.find(UpdateAssignedUser).exists()).toBeFalsy();
+    expect(wrapper.find(Dropdown.Item).at(2).text().includes(dropdownOptions[2])).toBeTruthy();
+    wrapper.find(Dropdown.Item).at(2).simulate('click');
+    expect(wrapper.find(UpdateAssignedUser).exists()).toBeTruthy();
+  });
+
+  it('Calls updateAssignedUsers and updateTable methods when component mounts', () => {
+    const instance = getInstance();
+    const updateAssignedUsersSpy = jest.spyOn(instance, 'updateAssignedUsers');
+    expect(updateAssignedUsersSpy).toHaveBeenCalledTimes(0);
+    instance.componentDidMount();
+    expect(updateAssignedUsersSpy).toHaveBeenCalledTimes(1);
+  });
+
+  describe('ExposureDashboard', () => {
+    it('Properly renders all tabs', () => {
+      const wrapper = getExposureWrapper();
+      for (var key of Object.keys(mockExposureTabs)) {
+        expect(wrapper.find('#' + key + '_tab').exists()).toBeTruthy();
+        expect(wrapper.find('#' + key + '_tab').text()).toEqual(mockExposureTabs[key]['label']);
+      }
+    });
+  });
+
+  describe('IsolationDashboard', () => {
+    it('Properly renders all tabs', () => {
+      const wrapper = getIsolationWrapper();
+      for (var key of Object.keys(mockIsolationTabs)) {
+        expect(wrapper.find('#' + key + '_tab').exists()).toBeTruthy();
+        expect(wrapper.find('#' + key + '_tab').text()).toEqual(mockIsolationTabs[key]['label']);
+      }
+    });
+  });
+});
```

Comment:
new line at end of file?

Reply:
Will make this adjustment with my next commit.